Core/MIPS/x86/X64IRCompALU.cpp

```diff
@@ -164,10 +164,28 @@ void X64JitBackend::CompIR_Bits(IRInst inst) {
 
 	case IROp::ReverseBits:
 	case IROp::BSwap16:
-	case IROp::Clz:
 		CompIR_Generic(inst);
 		break;
 
+	case IROp::Clz:
+		regs_.Map(inst);
+		if (cpu_info.bLZCNT) {
+			LZCNT(32, regs_.RX(inst.dest), regs_.R(inst.src1));
+		} else {
+			BSR(32, SCRATCH1, regs_.R(inst.src1));
+			FixupBranch notFound = J_CC(CC_Z);
+
+			MOV(32, regs_.R(inst.dest), Imm32(31));
+			SUB(32, regs_.R(inst.dest), R(SCRATCH1));
```

Comment:
I think you could change this to BSR directly into the target and XOR by 31 to reverse it instead? Would save the mov, which isn't exactly significant heh.